benchmark/e2e/Qwen3.5/run_blocks.py

```
@@ -0,0 +1,182 @@
+    # not a different model: cost = f * fwd + (fwd_bwd - fwd)
+    f = args.forwards_per_backward
+
+    def weighted(fwd, fwd_bwd):
+        return f * fwd + (fwd_bwd - fwd)
+
+    blocks = {
+        "linear_attn (GDN+conv+MLP)": (cfg["num_linear_layers"], weighted(*measured["linear_attn (GDN+conv+MLP)"])),
+        "full_attn (SDPA+MLP)": (cfg["num_full_attention_layers"], weighted(*measured["full_attn (SDPA+MLP)"])),
+    }
+    head_ms = head_fwd_bwd
+    print(f"\n  per block, fwd / fwd+bwd ms:")
```

Ruff (0.16.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)